### DISCUSSION TITLE
Downtime for search.xcloud.live

### DISCUSSION THREAD

**Discussion by TechMitten** [searxng/searx-instances#385] [Report instance downtime]
I'm having trouble with my server and my instance will be down until further notice.

**Comment by TechMitten** [comment]
My instance is back up and will remain this way. Sorry for the downtime.

**Comment by TechMitten** [comment]
Minutes after I run my instance again both Google and DuckDuckGo are suspended?

**Comment by unixfox** [comment]
Thank you for the notifying us.
